guides/common/modules/proc_configuring-ipxe-environment.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Configuring_iPXE_Environment_{context}"]
+= Configuring iPXE environment
+
+Configure iPXE environment on all {SmartProxies} that you want to use for iPXE provisioning.
+
+ifndef::foreman-deb[]
+.HTTP security consideration
+In {EL}, security-related features of iPXE are not supported and the iPXE binary is built without security features.
+For this reason, you can only use HTTP but not HTTPS.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/solutions/3483601[{RHEL} HTTPS support in iPXE].
+endif::[]
+ifndef::satellite[]
+You can recompile iPXE from source to use security features such as HTTPS.
```

Comment:
This consideration only applies to the iPXE firmware for chainbooting.
Can the user, hypothetically, use HTTPS for booting VMs if the hypervisor supports it?
cc @ekohl @stejskalleos

Reply:
Already clarified.